browser/iframe-browser.js

```diff
@@ -0,0 +1,77 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var CommandManager      = brackets.getModule("command/CommandManager"),
+        MainViewManager     = brackets.getModule("view/MainViewManager"),
+        Commands            = brackets.getModule("command/Commands"),
+        _panel		        = null,
+        horizontal          = true,
+        vertical            = false;
+
+
+
+    /*
+     * Function that manages initial arguments
+     * html: We are expecting the user to send us a url to the blob for use
+     *      if undefined we will not fill the iframe with an src
+     */
+    function browse(html) {
```

Comment:
If you're expecting a URL to be passed to you here, name your argument that: `function browser(url) {`